Source/WTF/Scripts/Preferences/WebPreferences.yaml

```diff
@@ -2553,10 +2553,13 @@ WebGPU:
   type: bool
   defaultValue:
     WebKitLegacy:
+      "ENABLE(WEBGPU)": true
```

Comment:
@litherum would it be acceptable to keep the flag in WebPreferences but behind an otherwise undefined variable that I can add to my LocalOverrides.xcconfig? This avoid some recompilation since for some reason modifying WebPreferences.yaml triggers a rebuild of a large number of files

Reply:
Yes, this solves the problem nicely.